src/main/java/pages/HomePage.java

```diff
@@ -1,9 +1,45 @@
 package pages;
 
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-//Header element (shopping cart button and menu button ) корзина. меню
+
 public class HomePage extends BasePage {
```

Comment:
HomePage should be abstract and you should inherit it in every other page-class which contains this menu and shoppingCart links. Then you could access these elements from any derived page.

Reply:
that is, in the HomePage class, I leave the elements. I create a class heir from it, for example, MenuDetailsPage, where I write methods, and already from it I take them into tests